include/micm/process/cuda_process_set.cuh

```diff
@@ -4,22 +4,37 @@ namespace micm
 {
   namespace cuda
   {
-    void AddForcingTerms_kernelSetup(
+   std::chrono::nanoseconds AddForcingTerms_kernelSetup(
```

Comment:
We've been trying to use CamelCase for function names (e.g., `AddForcingTermsKernelSetup`) here and below.

Also, I'm not a CUDA programmer but it was a little confusing to me that this function is called "setup" but also calls the kernel function. If this is a standard naming convention, I'm fine with that, but if not maybe consider something like`AddForcingTermsKernelDriver`. @sjsprecious - what do you think?

Reply:
changed it to AddForcingTermsKernelDriver for now. Let me know if Jian has any suggestion. @sjsprecious

Reply:
Thanks @mattldawson and @qinatan for pointing this out and fixing it. This is a good point as conventionally, only the code that actually runs on the GPU is called `CUDA kernel`. Other subroutines that prepare the input and output data are really `driver` subroutines as @mattldawson suggested. Changing the name makes it more clear now.